NAV_Algorithms/NAV_tuning_parameters.h

```diff
@@ -40,6 +42,7 @@
 #define DEFAULT_AVG_VARIO_TC 	30.0f
 #define DEFAULT_WIND_TC 	5.0f
 #define DEFAULT_WIND_AVG_TC 	30.0f
+#define NEGLECTABLE_WIND	0.1f
```

Comment:
The new parameter is not handled within the EEPROM, but the comment above indicates this.  Please add a new comment and separate the Non EEPROM parameters here.